ui-react/packages/atlasmap/src/UI/useInputTextSelectDialog.tsx

```diff
@@ -149,61 +157,60 @@ export function useInputTextSelectDialog({
       isFooterLeftAligned={false}
     >
       {textLabel1 && (
-        <InputGroup className={css(styles.iGroup)}>
-          <InputGroupText className={css(styles.iGroupTextLabel)}>
-            {textLabel1}
-          </InputGroupText>
+        <>
           <TextInput
-            key={"text-input-value1"}
+            className={css(styles.iGroupTextLabel)}
+            key={"itsd-text-input-value1"}
```

Comment:
Why the key? It should be used only when creating elements dynamically like with a map over an array

Reply:
corrected.